Cargo.toml

```diff
@@ -1,28 +1,29 @@
 [package]
-name = "bitcoinconsensus"
-version = "0.19.0-1"
-authors = ["Tamas Blummer <tamas.blummer@gmail.com>"]
+name = "zcashconsensus"
+version = "0.1.0"
+authors = ["Tamas Blummer <tamas.blummer@gmail.com>", "Zcash Foundation <zebra@zfnd.org>"]
 license = "Apache-2.0"
-homepage = "https://github.com/rust-bitcoin/rust-bitcoinconsensus/"
-repository = "https://github.com/rust-bitcoin/rust-bitcoinconsensus/"
-description = "Bitcoin's libbitcoinconsensus with Rust binding."
-keywords = [ "bitcoin", "bitcoinconsensus", "libbitcoinconsensus" ]
 readme = "README.md"
-exclude = ["/bitcoin/.github/**", "/bitcoin/.tx/**", "/bitcoin/build-aux/**", "/bitcoin/contrib/**", "/bitcoin/depends/**", "/bitcoin/doc/**", "/bitcoin/share/**", "/bitcoin/test/**", "/bitcoin/src/bench/**", "/bitcoin/src/config/**", "/bitcoin/src/consensus/**", "/bitcoin/src/leveldb/**", "/bitcoin/src/objtest/**", "/bitcoin/src/obj/**", "/bitcoin/src/policy/**", "/bitcoin/src/qt/**", "/bitcoin/src/rpc/**", "/bitcoin/src/support/**", "/bitcoin/src/test/**", "/bitcoin/src/univalue/**", "/bitcoin/src/wallet/**", "/bitcoin/src/zmq/**"]
 build = "build.rs"
+edition = "2018"
 
 [lib]
-name = "bitcoinconsensus"
+name = "zcashconsensus"
 path = "src/lib.rs"
 
 [features]
-external-secp = []
```

Comment:
I'm not sure if this was intentionally removed (it's still referenced in the build script, and seems like a useful feature to keep around as long as it works).